ci-operator/config/code-ready/crc/code-ready-crc-master.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
```

Comment:
This config does not specify a OpenShift release to be installed for the e2e tests, either via `tag_specification` or `releases` stanza. Please see the https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help/ci-operator#release.